### DISCUSSION TITLE
The support settings file cannot be deleted, but is it readable and writable? What do I need to do if I need support?

### DISCUSSION THREAD

**Discussion by 1247862674** [trapexit/mergerfs#1368] [General]

**Comment by trapexit** [comment]
I'm sorry but I don't understand what you're asking.

**Comment by 1247862674** [comment]
Is there any policy that can control files that cannot be deleted but can be read and written?

**Comment by 1247862674** [comment]
I now have a data source and I want to make it readable and writable, but no one else can delete the file

**Comment by trapexit** [comment]
Policies select files to act on. They don't control what can be read, written, or unlinked.

You need to be more specific about what you want to happen. What about creates?

**Comment by 1247862674** [comment]
For example, I now mergefs A:B C. I want A to be readable and writable, but all files in A cannot be deleted.

**Comment by trapexit** [comment]
Yes, I understand that but what about create? These concepts are orthogonal. You can read anything readable and has nothing to do with unlinking or creating or writing. Why not just use ACLS?

**Comment by 1247862674** [comment]
it can create

**Comment by trapexit** [comment]
Why don't you just use ACLs?

**Comment by 1247862674** [comment]
Are there any specific examples or documentation?

**Comment by 1247862674** [comment]
To be honest, I don’t know how to use ACLS.

**Comment by trapexit** [comment]
You aren't really explaining the full requirement so it is not possible for me to help. I would suggest just reading about unix and posix permissions and acls. It is fundamental to filesystems. If you don't want people to delete a file just don't give them permissions to it.
